docs/getting_started.rst

```diff
@@ -280,8 +279,8 @@ teracy-dev Git Clone and Vagrant Up
 
        $ cd ~/
        $ git clone https://github.com/teracyhq/dev.git teracy-dev
+       $ git checkout v0.6.0-a5
```

Comment:
We have to cd repo first before checkout branch

Reply:
@datphan I've updated! Please check again! Thank you